src/web/components/TopMenuBar.tsx

```diff
@@ -0,0 +1,78 @@
+import { Text } from '@fluentui/react-components';
+import styled from 'styled-components';
+
+function TopMenuBar() {
+
+
+// Return the top menu bar with the FinOps toolkit logo
+
+  return (
+    <FullWidthContainer>
+      <StyledCommandBar>
+        <LogoTextContainer>
+          <Logo src="logo-windows.png" alt="Microsoft Logo" />
+          <TextContainer>
+            <Text size={300} weight="medium">FinOps toolkit</Text>
+          </TextContainer>
+        </LogoTextContainer>
+
+      </StyledCommandBar>
+    </FullWidthContainer>
+  );
+}
+
+
+// Styled components for layout and customization
+const StyledCommandBar = styled.div`
```

Comment:
I feel like putting CSS in TSX is an antipattern. What do others think?

Reply:
This is what I started with when converting to CSS. These are styled components. I will convert to fluent ui if we get the same look and feel as a result. Then, we need to check if we can create global styles for components and call them out. I'm more than happy to change this and get input from others

Reply:
Fluent uses Griffel and the makestyles is something I am using now for the finops home page. It is not here in this first pr, but we can have a working session to go through this and then I can adapt accordingly.

Reply:
New commit contains only fluent v2 with makeStyles